terms.html

```diff
@@ -157,6 +157,8 @@
 is an authentic and timely statement of the issuer or presenter, respectively.  
     This includes checking that: the credential (or presentation) conforms to the specification; the proof method is
 satisfied; and, if present, the status check succeeds.
+Verification of a credential does not imply evaluation of the the truth content
+of <a>claims</a> encoded in the credential.</a>.
```

Comment:
```suggestion
Verification of a credential does not imply evaluation of the truth of the content
of <a>claims</a> encoded in the credential.</a>.
```

Reply:
I can live with @msporny's suggestion, but I think this phrasing is a bit less convoluted --
```suggestion
Verification of a credential does not imply evaluation of the truth
of <a>claims</a> encoded in the credential.</a>.
```

Reply:
I like @TallTed's suggestion better.

Reply:
+1 for @TallTed's suggestion.

Reply:
Committed @TallTed's suggestion: ec0793b23443715cfa67738a87386f407bc18d7c